SonarQube.Common/Serializer.cs

```diff
@@ -43,9 +43,11 @@ public static void SaveModel<T>(T model, string fileName)
             settings.NamespaceHandling = NamespaceHandling.OmitDuplicates;
             settings.OmitXmlDeclaration = false;
 
-            using (XmlWriter writer = XmlWriter.Create(fileName, settings))
+            using (MemoryStream stream = new MemoryStream())
```

Comment:
Please add a comment explaining your intent here, as it wouldn't be clear to someone reading the code.